bin/ensure_config.sh

```diff
@@ -0,0 +1,3 @@
+#! /usr/bin/env bash
+
+jira-example-config --install
```

Comment:
So referencing the comment above, my guess is that this command could fail because a config is already installed.
In that case, I agree with @brolewis that this script should handle creating the correct return/exit status.
Maybe if `jira-example-config` had an `--install-if-missing` or some such, then this script wouldn't be needed and travis could call that directly? thinking out loud.